db/seeds.rb

```diff
@@ -49,14 +49,24 @@
 
   # Could have done in the above loop, but that was getting a bit messy
   puts 'Creating Items in the API, by Item Type'
+  items = YAML::load_file(File.join(Rails.root, 'db/db_yml', 'items.yml'))
+  itemsint = 0
```

Comment:
1. Please switch to snake_case in the future
2. Do you need this? The `i` should work, no?